website/docs/guides/migration/versions/02-upgrading-to-v1.5.md

```diff
@@ -38,6 +38,9 @@ Changes planned for v1.5:
 - Renaming ["global configs"](global-configs) for consistency ([dbt-core#6903](https://github.com/dbt-labs/dbt-core/issues/6903))
 - Moving `log-path` and `target-path` out of `dbt_project.yml` for consistency with other global configs ([dbt-core#6882](https://github.com/dbt-labs/dbt-core/issues/6882))
 - As described in [dbt-core#7169](https://github.com/dbt-labs/dbt-core/pull/7169), command-line parameters that could be silent before will no longer be silent. See [dbt-labs/dbt-core#7158](https://github.com/dbt-labs/dbt-core/issues/7158) and [dbt-labs/dbt-core#6800](https://github.com/dbt-labs/dbt-core/issues/6800) for more examples of the behavior we are fixing.
+- `dbt list` will no longer suppress `INFO` level logs. To pipe the results of dbt list to a file or another process, and exclude unrelated log messages from the output, you can use either of the following flags:
```

Comment:
Suggesting turning this into a what they can do now. Also I'm not clear what 'pipe the results' mean so I also suggested to clarify it, but again I'm not sure if pipe is a known software eng term. 

Also i think we use flags/commands - sugg using commands since the dbt list doc says command and is under the command section. 

```suggestion
- `dbt list` command will display `INFO` level logs, previously it suppressed it. To pipe (or connect) the results of dbt list to a file or another process, and exclude unrelated log messages from the output, you can use either of the following commands:
```

Reply:
`Pipe` means to take the output of a command and redirect it to a file (such as a text file). It's a fairly common expression for those working in command line tools and should be familiar to much of our user base.

Reply:
I wanted to clarify dbt list will show more than just INFO. How does this sound:
```suggestion
- `dbt list` command will now include `INFO` level logs by default; previously, they were suppressed. To pipe the results of `dbt list` to a file or another process and exclude unrelated log messages from the output, you can use either of the following commands:
```